modules/ci_environment/manifests/jenkins_job_support/postgresql.pp

```diff
@@ -10,4 +10,16 @@
       password_hash => postgresql_password('jenkins', 'jenkins'),
       createdb      => true;
   }
+
+  # For mapit
+  class { 'postgresql::server::postgis': }
+
+  # The mapit role needs to be a superuser in order to load the PostGIS
+  # extension for the test database.
+  postgresql::server::role {
+    'mapit':
+      superuser     => true,
+      password_hash => postgresql_password('mapit', hiera('mapit::postgres_role_password'));
```

Comment:
I think the password could be passed in as a parameter to this class just below line 3 as something like `$mapit_password` which would avoid the call to `hiera()` from this line. A similar example is [`ci_environment::github_gds_cert`](https://github.com/alphagov/ci-puppet/blob/4b4af1c46336d2ab7fec00567c56a0426a6fcb50/modules/ci_environment/manifests/github_gds_cert.pp#L5-L7).